app/assets/json/translations/react/en.json

```diff
@@ -37,6 +37,14 @@
   "accountSettings.verifyEmail": "Please check your email inbox to verify this change.",
   "alreadySubmitted.message": "You have already submitted an application to this listing.",
   "alreadySubmitted.title": "Good news!",
+  "assistance.contact.helpLine.title1": "For help with San Francisco services 24 hours a day, 7 days a week:",
+  "assistance.contact.helpLine.subtitle": "Call 311 or <a target='_blank' href='https://sf311.org/'>visit the 311 website.</a>",
+  "assistance.contact.questionsAboutListings.title1": "For questions about DAHLIA listings:",
+  "assistance.contact.questionsAboutListings.subtitle": "Contact the agent shown on the listing page.",
+  "assistance.contact.questionsAboutPrograms.title1": "For questions about housing programs:",
+  "assistance.contact.questionsAboutPrograms.contactWebsite": "<a target='_blank' href='https://sf.gov/departments/mayors-office-housing-and-community-development'>Visit the website for the Mayor’s Office of Housing and Community Development </a>(MOHCD)",
```

Comment:
@yindipei or @emilyshore do we have language specific links for the sf.gov pages?

Reply:
Good call! The link should be different depending on what language page the user is on:
Spanish MOHCD page: https://sf.gov/es/node/55
Chinese MOHCD page: https://sf.gov/zh-hant/node/55
Filipino MOHCD page: https://sf.gov/fil/node/55